### ISSUE TITLE
`Get-AzSqlDatabaseReplicationLink` errors out when not specifying `-PartnerServerName`

### ISSUE THREAD

**Issue by isra-fel** [Azure/azure-powershell#10485]
## Description

One of our customer ran `Get-AzSqlDatabaseReplicationLink` without specifying `-PartnerServerName`, he got an error:
![PS screenshot](https://user-images.githubusercontent.com/17446043/68262248-50967f00-0008-11ea-9be0-625568ab9cec.png)

Original issue: #10350 

## Steps to reproduce

```powershell
Get-AzSqlDatabaseReplicationLink -ResourceGroupName "rg1" -ServerName "sv1" -DatabaseName "db1" -PartnerResourceGroupName "prg1"
```

## Root cause
https://github.com/Azure/azure-powershell/blob/master/src/Sql/Sql/Replication/Cmdlet/GetAzureSqlDatabaseReplicationLink.cs#L76
```cs
if (MyInvocation.BoundParameters.ContainsKey(PartnerServerName) && !WildcardPattern.ContainsWildcardCharacters(PartnerServerName))
```
`ContainsKey(PartnerServerName)` should be `ContainsKey(nameof(PartnerServerName))`

## Module versions

```powershell
Az.Sql 1.15.0 and 2.0.0 both have the issue.
```

**Comment by isra-fel** [comment]
## Suggested work items

[ ] Fix the error
[ ] Add a test case that cover when `-PartnerServerName` is not specified
[ ] Do a search in the module to find other possible similar mistakes

Thanks

**Comment by SatishBoddu-MSFT** [comment]
Hello @isra-fel @maboja-msft @jaredmoo Please let us know if we are good to close this issue? Please let us know if additional help is needed in this matter?

**Comment by isra-fel** [comment]
After searching I found it's already fixed by #13433
Closing now.
